pkg/api/defaults-sysctld.go

```diff
@@ -14,6 +14,7 @@ func (cs *ContainerService) setSysctlDConfig() {
 		"net.ipv4.neigh.default.gc_thresh1": "4096",
 		"net.ipv4.neigh.default.gc_thresh2": "8192",
 		"net.ipv4.neigh.default.gc_thresh3": "16384",
+		"kernel.dmesg_restrict":             "0",
```

Comment:
You'll have to update the UT test cases in the `TestSetSysctlDConfig` test func in `pkg/api/defaults-sysctld_test.go` to incorporate this addition. We assume this is a safe addition for 18.04 as well (in the next few months while it's 18.04 is still a thing)?

Reply:
Yes, this is safe for 18.04 as it is the default in 18.04 - 20.04 changed the default and this was the best way to keep them compatible.  (We depend on this to be able to debug problems in GPU drivers and such)